src/interfaces/wallet.cpp

```diff
@@ -275,6 +279,7 @@ WalletTxStatus MakeWalletTxStatus(AnonWallet* pAnonWallet, const uint256 &hash,
 //! Construct wallet TxOut struct.
 WalletTxOut MakeWalletTxOut(CWallet& wallet, const CWalletTx& wtx, int n, int depth)
 {
+    LOCK(cs_main);
```

Comment:
Likewise, all the callers to this have cs_main locked already as well.

Reply:
that looks right.